### PR TITLE
README, added Usage:defualts

### PR DESCRIPTION
**Pull request by tomByrer**
Just to inform users what `package.json` fields are consumed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,9 +62,20 @@ module.exports = function (grunt) {
 ```
 
 
-## Usage examples
+## Usage
 
-#### mixins
+#### defaults
+
+Atleast the following fields in `package.json` are used (in precident):
```
Comment (jonschlinkert) [inline README.md:69]: Sorry to nitpick, would you mind changing this line to

"The following properties in `package.json` are used:"

Comment (jonschlinkert) [inline README.md:69]: actually, nvm, I'm going to update the README with more copyright info anyway, I'll merge this first. thanks!

**Comment by tomByrer** [comment]
> I'm going to update the README with more copyright info

Great idea, came out nice.
:sigh: 2013 is already an _old_ year.

**Comment by jonschlinkert** [comment]
thanks!

**Pull request merged**